src/main/java/uk/gov/pay/products/model/Product.java

```diff
@@ -118,10 +137,11 @@ public static Product from(JsonNode jsonPayload) {
         Boolean referenceEnabled = (jsonPayload.get(FIELD_REFERENCE_ENABLED) != null) && jsonPayload.get(FIELD_REFERENCE_ENABLED).asBoolean();
         String referenceLabel = (jsonPayload.get(FIELD_REFERENCE_LABEL) != null) ? jsonPayload.get(FIELD_REFERENCE_LABEL).asText() : null;
         String referenceHint = (jsonPayload.get(FIELD_REFERENCE_HINT) != null) ? jsonPayload.get(FIELD_REFERENCE_HINT).asText() : null;
+        SupportedLanguage language = (jsonPayload.get(FIELD_LANGUAGE) != null) ? SupportedLanguage.fromIso639AlphaTwoCode(jsonPayload.get(FIELD_LANGUAGE).asText()) : SupportedLanguage.ENGLISH;
```

Comment:
I suppose an alternative might be something like this:

```java
SupportedLanguage language = Optional.ofNullable(jsonPayload.get(FIELD_LANGUAGE)).map(JsonNode::asText).map(SupportedLanguage::fromIso639AlphaTwoCode).orElse(SupportedLanguage.ENGLISH);
```

But consistency is good too

Reply:
I have made this change, it is nicer